docs/source/index.rst

```diff
@@ -523,8 +522,10 @@ Polling is not subject to handle state with HTTP status codes (e.g. 304)
                        ceph.conf sections (only global, mon, osd, rgw, mds
                        allowed) to keys and values. Anything defined in this
                        mapping will override existing settings.
-   :<json object devices: (required) A mapping of OSD device to Journal
-                          like device: {"device": "journal"}.
+   :<json object or array devices: (required) A mapping of OSD device to Journal
+                          like device: {"device": "journal"} (when the journal
+                          is separate from the OSD) or ["/dev/sdb", "/dev/sdc"]
+                          for collocated OSDs and Journals.
```

Comment:
@japplewhite If you're wanting to use the same device for the journal for many OSDs then you'll want to use dedicated journals. Here's the documentation on that: http://docs.ceph.com/ceph-installer/docs/#journals

As an example:
```
"devices": {"/dev/sdd": "/dev/sdc", "/dev/sde": "/dev/sdc"}
```